tests/components/homekit/test_accessories.py

```diff
@@ -70,6 +70,7 @@ def test_home_accessory(self):
         acc = HomeAccessory(hass, 'Home Accessory', 'homekit.accessory', 2, {})
```

Comment:
Subsequently this could be `homekit.acc`

Reply:
Done.